app/controllers/api/v1/jwt_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::V1Controller < ApplicationController
+    include JWTHelper
+    skip_before_action :verify_authenticity_token
+
+    def jwt
+        params[:login] = {"login"=>params[:username], "password"=>params[:password], "remember_me"=>"0"}
```

Comment:
So here I'm not accepting  :login parameter, I'm creating it from the :username and :password parameters that I'm accepting and just putting it into the format that the authenticate method accepts.

Reply:
You will need to specify what params can be passed into the api.  What if they pass in invalid parameters or no parameters?  You’re going to get an error message or allow risky parameters to be accepted by the api.  You can format the parameters to pass onto other objects as you wish.  But you will need to specify the permitted parameters, so that we are not creating a security risk.